### DISCUSSION TITLE
DacFx updates - how do you want to hear about them?

### DISCUSSION THREAD

**Discussion by dzsquared** [microsoft/DacFx#26] [Q&A] [answered]
With full updates to DacFx coming a few times a year, what would be your preferred method for being notified of a new release?  

The NuGet package is updated regularly with preview and GA versions.  GA versions of SqlPackage.exe and DacFx.msi are posted to the Microsoft docs page, but we haven't been 📢 promoting them as much as we could.

Some possible ideas:
- blog posts on SQL Server blog
- GA releases on this repository
- social media post from AzureSQL/SQLServer accounts

What are your suggestions or preference?

**Comment by ErikEJ** [comment, marked as answer]
Any kind of release notes would be a huge benefit over the current silence.

At a minimum GA release notes in this repo.

And maybe blog post and social media if there is something major?
